fetch_cluster_logs.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+#
+# SUMMARY:
+# Helper script that can fetch all logs for any cluster and outputs them in the current working directory.
+# Fetched logs are comprised of journald logs, mesos logs, sandbox logs and diagnostics bundles for all nodes.
+#
+# USAGE:
+# ./fetch_cluster_logs <master_public_ip> <ssh_user> --dcos_login_token=<dcos_login_token> --identity_file=<identity_file> --debug
+
+set +e
+set -x
+
+master_public_ip=$1
+if [[ -z $master_public_ip ]]; then
+  echo "ERROR: Required argument 'master_public_ip' was either not specified or null."
+  exit 0
+fi
+
+ssh_user=$2
+if [[ -z $ssh_user ]]; then
+  echo "ERROR: Required argument 'ssh_user' was either not specified or null."
+  exit 0
+fi
+
+for i in "$@"
+do
+case $i in
+    --debug)
+    # turn on the debug logging for the dcos cli
+    debug_options="--debug --log-level=debug"
+    shift
+    ;;
+    --dcos_login_token=*)
+    dcos_login_token="${i#*=}"
+    shift
+    ;;
+    --identity_file=*)
+    identity_file="${i#*=}"
+    shift
+    ;;
+esac
+done
+
+ssh_options="-A -T -l $ssh_user -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
+if [[ ! -z $identity_file ]]; then
+  ssh_options="-i $identity_file ${ssh_options}"
+fi
+
+# download the dcos-cli
+if [ ! -f dcos-cli ]; then
+  wget https://downloads.dcos.io/binaries/cli/linux/x86-64/dcos-1.12/dcos --output-document=dcos-cli
```

Comment:
Just a note: `curl` is more universally available than `wget`. 
If we have tested this in all our platforms and wget is available, then keeping this as such is fine. Otherwise, I would have gone with `curl -o`.

Reply:
https://curl.haxx.se/docs/manpage.html#-o